index.html

```diff
@@ -36,17 +33,22 @@ <h3>Greater Miami Valley Computer Science Circle</h3>
 
       <div id="page_content" style="left: 0px; top: 0px; width: 840px">
         <div id="content" style="left: 4px; top: -1px; width: 495px">
-          <img alt="" height="222" src="images/gmvcsc.jpg" width="481" />
-          <h2>Welcome to the GMVMC</h2>
-          <p>The Greater Miami Valley Computer Science Circle is here to help you become
-            better at Computer releated problem solving and thinking.&nbsp; We have
-            Computer Circle&nbsp; meetings twice a month to help High School and
-            Advanced middle school students.</p>
+          <h2>Welcome to the GMVCSC</h2>
+          <p>The Greater Miami Valley Computer Science Circle (GMVCSC), in its 3rd year of operation, is an organization dedicated to helping the students of the Miami Valley and beyond. Our Computer circle focuses on computational thinking and problem solving. We use Python/Java/HTML to teach concepts like Algorithm design, Web design, Mobile apps, Robotics, and Information Security. We currently only have 1 class level, as we get more instructors we will try to break out into beginning and intermediate/advanced classes. The GMVCSC is open to 7-12 graders who have had at least Algebra 1. Faculty come from local school districts, area colleges and universities, private industry, local government. We have students from all over the area and beyond; we are truly a “Greater Miami Valley” organization!
+We meet from 10:00am-Noon the 1st and 3rd Saturday of the month during the school year (September-early May) in the Mathematical and Microbiological Sciences (MM) Building on the campus of Wright State University.
+
+If you would like to inquire about joining one of our groups, please contact either Mr. Ken Nelson (at ken.nelson@wright.edu) or Dr. Munsup Seoh (at munsup.seoh@wright.edu). Spaces are limited! If there is not space for your child at the time you email us, then you will be placed on our waiting list. Available spaces will be offered to those on our waiting list first, with priority given to those on the list longest.

```

Comment:
You might want to wrap each paragraph in its own `<p>`...`</p>` tags. Within a paragraph tag, blank lines are ignored.

Reply:
Louis, go ahead and make this change:

The Greater Miami Valley... is a new paragraph.
We mee from 10:00am... is another paragraph.
If you would like... is another paragraph.
If you or some you know... is another paragraph.
Lastly... is another paragraph.
Coordinators: is another paragraph.